spec/services/adviser/application_form_validations_spec.rb

```diff
@@ -0,0 +1,211 @@
+require 'rails_helper'
+
+RSpec.describe Adviser::ApplicationFormValidations, type: :model do
+  let(:application_form) { create(:application_form) }
+  let(:candidate) { application_form.candidate }
+
+  subject(:validations) { described_class.new(application_form) }
+
+  describe 'delegations' do
+    it { is_expected.to delegate_method(:email_address).to(:candidate) }
+    it { is_expected.to delegate_method(:id).to(:application_form) }
+    it { is_expected.to delegate_method(:first_name).to(:application_form) }
+    it { is_expected.to delegate_method(:last_name).to(:application_form) }
+    it { is_expected.to delegate_method(:date_of_birth).to(:application_form) }
+    it { is_expected.to delegate_method(:phone_number).to(:application_form) }
+    it { is_expected.to delegate_method(:country).to(:application_form) }
+    it { is_expected.to delegate_method(:postcode).to(:application_form) }
+    it { is_expected.to delegate_method(:maths_gcse).to(:application_form) }
+    it { is_expected.to delegate_method(:english_gcse).to(:application_form) }
+    it { is_expected.to delegate_method(:science_gcse).to(:application_form) }
+  end
```

Comment:
Potentially not needed as it's testing implementation rather than functionality.  Can be removed.